plantuml2freemind/custom_types.py

```diff
@@ -1,4 +1,111 @@
-from typing import NewType, Dict, Any
+from typing import Any, Container, Dict, List, Optional
 
-# TODO: add real type: attrs is a good choice
-MindmapTreeType = NewType('MindmapTreeType', Dict[str, Any])
+from attr import asdict, attrib, attrs
+from attr.validators import in_, instance_of, optional
+
+LEFT_SIDE = 'left'
+RIGHT_SIDE = 'right'
+VALID_SIDES = frozenset([LEFT_SIDE, RIGHT_SIDE])
+
+
+def exclude_keys(d: Dict[str, Any], excluded: Container[str]) -> Dict[str, Any]:
+    return {key: value for key, value in d.items() if key not in excluded}
+
+@attrs
+class MindmapTreeNode(object):
```

Comment:
It's not an actual change request, but personally I think that docstring for each data structure class is a very useful thing. Especially with examples and explanations about the existence of structures in very this form.